pkgs/desktops/plasma-5/plasma-workspace/default.nix

```diff
@@ -46,6 +46,18 @@ mkDerivation {
   patches = [
     ./0001-startkde.patch
     ./0002-absolute-wallpaper-install-dir.patch
+    ( # Systemd v246 hides the interface used by KDE to fetch the
+      # user session details.
+      # We're applying this commit from 5.19.90 upgrading the
+      # systementry applet to use the new sessionmanagement API.
+      # See https://github.com/NixOS/nixpkgs/issues/98141 for more
+      # details.
+      # /!\ Remove this patch for version >= v5.19.90
+      fetchpatch {
+      name = "0003-port-systementry-to-sessionmanagement-api.patch";
+      url = "https://invent.kde.org/plasma/plasma-workspace/-/commit/05414ed58d43d87d907326636faac53ae2e7bd60.patch";
+      sha256 = "16izfcwxjkdn99j777ywjkzl01iyl542h4hpsbpkckccj7hz8sin";
+    })
```

Comment:
This may be hacky since there's not a good "Qt source of truth", but would we be able to conditionally disable this, something like:
```
] ++ lib.optional (lib.versionOlder qtquickcontrols.version "5.19") [
  <patch>
 ];
```

Reply:
actually, the Qt PRs will probably visit this, and might not be necessary.

Too much time spent doing python PRs :)